shells/firefox/src/contentScript.js

```diff
@@ -46,14 +46,19 @@ function connectToBackend() {
 
 function injectBackend() {
   var node = document.createElement('script');
+  var documentElement = document.documentElement;
 
   node.onload = function() {
     window.postMessage({source: 'react-devtools-reporter'}, '*');
 
     connectToBackend();
-    node.parentNode.removeChild(node);
+    if (node.parentNode) {
```

Comment:
It should be impossible to not have a parent when `onload` is called; throwing an error would probably be more readable than adding a seemingly-unnecessary condition check to the code to just make flow happy.